.gitmodules

```diff
@@ -15,5 +15,5 @@
 	shallow = true
 [submodule "HiGHS"]
 	path = scipy/_lib/highs
-	url = https://github.com/scipy/highs
+	url = https://github.com/mckib2/highs
```

Comment:
TODO: change back (and update submodule commit) when [scipy/HiGHS ](https://github.com/scipy/HiGHS/pull/59) merges